src/xdp-app-info.c

```diff
@@ -175,6 +180,11 @@ xdp_app_info_set_property (GObject      *object,
       priv->instance = g_value_dup_string (value);
       break;
 
+    case PROP_PIDFD:
+      g_assert (priv->pidfd == -1);
+      priv->pidfd = dup (g_value_get_int (value));
```

Comment:
`typedef enum _XdpAppInfoFlags`

tbh, I'm unsure why this even is a pattern in e.g. mutter but at least it makes it easy to grep for the declaration...

Reply:
It's nice because one can use ctags to go directly to the actual struct, instead of the typedef, by adding a `_` in front of the name.